src/common/interfaces/webchat-config.ts

```diff
@@ -19,6 +19,7 @@ export interface IWebchatSettings {
     enableConnectionStatusIndicator: boolean;
     showEngagementMessagesInChat: boolean;
     enableFileUpload: boolean;
+    enableFocusTrap: boolean;
```

Comment:
you will also need to add an entry to `docs/embedding.md` explaining what this flag does

Reply:
Fixed